src/screens/inventory.py

```diff
@@ -0,0 +1,144 @@
+import pygame  # noqa
+from src.gui.abstract_menu import AbstractMenu
+from src.enums import FarmingTool, InventoryResource, GameState
+from src.gui.components import ImageButton
+from src.settings import SCREEN_WIDTH, SCREEN_HEIGHT
+from itertools import chain
+
+
+class _IMButton(ImageButton):
+    def __init__(self, content: pygame.Surface, rect: pygame.Rect, name: str):
+        super().__init__(content, rect)
+        self._name = name
+
+    @property
+    def text(self):
+        return self._name
+
+
+_SPACING_BETWEEN_ROWS = 20
+_TOP_MARGIN = 160
+_LEFT_MARGIN = 40
+_BUTTON_SIZE = (80, 80)
+_SECTION_TITLES = (
+    "Resources",
+    "Tools",
+    "Special Items"
+)
+
+
+class InventoryMenu(AbstractMenu):
+    _IR_TO_OVERLAY_IMG = {
```

Comment:
If we add a new ressource in enum this will not update with

Reply:
That's a good catch